manifests/init.pp

```diff
@@ -217,33 +203,21 @@
     }
   }
 
-  if $puppet {
-    class { '::certs::puppet':
-      hostname => $foreman_proxy_fqdn,
-    } ~>
-    class { '::puppet':
-      server                      => true,
-      server_ca                   => $::foreman_proxy::puppetca,
-      server_foreman_url          => $foreman_url,
-      server_foreman_ssl_cert     => $::certs::puppet::client_cert,
-      server_foreman_ssl_key      => $::certs::puppet::client_key,
-      server_foreman_ssl_ca       => $::certs::puppet::ssl_ca_cert,
-      server_storeconfigs_backend => false,
-      server_dynamic_environments => true,
-      server_environments_owner   => 'apache',
-      server_config_version       => '',
-      server_enc_api              => 'v2',
-      server_ca_proxy             => $puppet_ca_proxy,
-      server_implementation       => $puppet_server_implementation,
-      additional_settings         => {
-                                        'disable_warnings' => 'deprecations',
-      },
-    }
+  # We can't pull the certs out to the top level, because of how it gets the default
+  # parameter values from the main ::certs class.  Kafo can't handle that case, so
+  # it remains here for now.
+  class { '::certs::puppet':
```

Comment:
To be more specific, this block here -- https://github.com/Katello/puppet-foreman_proxy_content/pull/108/files#diff-60ae41fd0a31977447947f59940ee9a4R213